.github/workflows/ci-app-cloud-e2e-test.yml

```diff
@@ -53,6 +53,7 @@ jobs:
           - collect_failures
           - custom_work_dependencies
           - drive
+          - drive_mounted_s3
```

Comment:
We are moving the e2e to [azure piplelines](https://github.com/Lightning-AI/lightning/pull/14282). You might have to add this to the new CI config file

Reply:
cc @Borda

Reply:
yep, let's see which PR gets merged first... :)

Reply:
updated